distributed/cli/dask_scheduler.py

```diff
@@ -46,8 +43,18 @@
               help="File to write connection information. "
               "This may be a good way to share connection information if your "
               "cluster is on a shared network file system.")
+@click.option('--certfile', default=config.get('tls-certfile'),
+              help='path to certfile to use for ssl connection')
+@click.option('--keyfile', default=config.get('tls-keyfile'),
+              help='path to keyfile to use for ssl connection')
+@click.option('--default-scheme', type='str', default=config.get("default-scheme"),
+              "Default comminiocation scheme to use.")
```

Comment:
Running tests locally shows that this needs a keyword name, like `help="Default communication ..."` (also you have a prose typo).